ark/utils/spatial_analysis_utils_test.py

```diff
@@ -74,6 +75,15 @@ def make_example_data_closenum():
     dist_mat[8, 7] = 50
     dist_mat[7, 8] = 50
 
+    # add some randomization to the ordering
+    coords_in_order = np.arange(dist_mat.shape[0])
+    coords_permuted = deepcopy(coords_in_order)
+    np.random.shuffle(coords_permuted)
+
+    # we have to 1-index coords because people will be labeling their cells 1-indexed
+    dist_mat = xr.DataArray(dist_mat[np.ix_(coords_permuted, coords_permuted)],
```

Comment:
I think it's good practice for xarrays that whenever we're creating coords or dims, if it's not directly equal to something else, it happens on its own line
so:
````
coords=[coords_permuated + 1, coords_permuated + 1]
dist_mat = xr.DataArray(values, coords=coords)